.github/workflows/test.yml

```diff
@@ -67,13 +131,12 @@ jobs:
         curl -fSsLl $REPO/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/devel_tools_criu.gpg > /dev/null
         echo "deb $REPO/ /" | sudo tee /etc/apt/sources.list.d/criu.list
         sudo apt update
```

Comment:
this `sudo apt update` needs to be removed as we now have a step above that does it

Reply:
No, the source list is updated after that